api/upload/uploadModel.js

```diff
@@ -4,6 +4,13 @@ const add = async (data) => {
   return await db('cases').insert(data);
 };
 
+const update = async (data) => {
```

Comment:
Let's discuss this function this am after stand up. I think we'll need the same functionality to update from 'review' to 'pending'. We can either create to similar functions with different statuses or we can refactor this one and pass in the status as an argument.

The answer to this may be that the review will take place elsewhere (perhaps the caseRouter and caseModel).

Reply:
Sure, that will be the exact same function. Can just add a status argument.